__DEFINES/setup.dm

```diff
@@ -1371,6 +1371,7 @@ var/proccalls = 1
 #define FOOD_LIQUID	8
 #define FOOD_SKELETON_FRIENDLY 16 //Can be eaten by skeletons
 #define FOOD_LACTOSE 32 //Contains MILK
+#define FOOD_NEEDS_COLD 32	//Food needs to be frozen/refriderated or it will rot.
```

Comment:
This won't work. These are bitflags, they represent a binary bit. FOOD_LACTOSE is already 32 (aka 00001) so FOOD_NEEDS_COLD needs to be 64 (aka 000001) so they don't overlap. Otherwise you'd have no way to know if a food with the 5th bit set to 1 means it contains lactose or needs cold.

Reply:
local tired man finally realizes why they were getting doubled, Thanks as usual Raven!